README.md

```diff
@@ -92,7 +92,9 @@ render(<App />, document.getElementById('root'));
 
 ### For development
 
-1. At the project root, run `yarn install`
+1. Fork this repository using by clicking _Fork_ button at the top
+1. Clone your own version on repository by using `git clone https://github.com/<username>/reactfire.git` (replace the repo url with your own version's url)
```

Comment:
I think we can simplify the first two down to:

```md
1. [Clone](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) this repository (or a [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo#propose-changes-to-someone-elses-project))
```

Which will look like this:

1. [Clone](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) this repository (or clone a [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo#propose-changes-to-someone-elses-project))